src/pickit.py

```diff
@@ -47,28 +52,36 @@ def pick_up_items(self, char: IChar) -> bool:
                         closest_item = item
                 x_m, y_m = self._screen.convert_screen_to_monitor(closest_item.center)
                 if closest_item.dist < self._config.ui_pos["item_dist"]:
-                    # no need to stash potions, scrolls, or gold 
+                    # no need to stash potions, scrolls, or gold
                     if (("potion" not in closest_item.name) and ("tp_scroll" != closest_item.name) and ("misc_gold" not in closest_item.name)):
                         found_items = True
+
+                    #if picking up potion, then add to potions_remaining
+                    if ("potion" in closest_item.name):
+                        predictPotCol=self._belt_manager.predict_pot_col(closest_item.name)
+                        #Logger.debug(f"predictPotCol={predictPotCol}")
+                        if predictPotCol >= 0:
+                            self._belt_manager.potions_remaining[predictPotCol] = self._belt_manager.potions_remaining[predictPotCol] + 1
```

Comment:
You have to be a bit careful here. Pickit might "see" the same potion multiple times.